hq_validation_template.md

```diff
@@ -1,26 +1,22 @@
 # Validation template for High Quality tag request
 
-Package URL: 
+Package URL:
 
-This template is designed to be used as it is by Apps group to validate requests from packagers for the tag High Quality.
+This template is designed to be used by the Apps group to validate requests from packagers for the tag High Quality.
 
-Mandatory check boxes:
-- [ ] The package is level 7.
-- [ ] The package has been level 7 for at least 1 month.
-- [ ] The package has been in the list for at least 2 months.
```

Comment:
Do we really want to drop this? It allows us to know if the packager will maintain its app seriously. (But at the same time, if he makes the effort to make the request, he is normally motivated to stay).

Reply:
The idea is that basically this is now part of level 8 (c.f. the "long term quality" check)